protocols/gossipsub/src/behaviour.rs

```diff
@@ -668,6 +632,51 @@ where
                         .cloned(),
                 );
             } else {
+                match self.mesh.get(&raw_message.topic) {
+                    // Mesh peers
+                    Some(mesh_peers) => {
+                        recipient_peers.extend(mesh_peers);
+                    }
+                    // Gossipsub peers
+                    None => {
+                        tracing::debug!(topic=%topic_hash, "Topic not in the mesh");
+                        // If we have fanout peers add them to the map.
+                        if self.fanout.contains_key(&topic_hash) {
```

Comment:
This could be a `match`-arm guard of `None` which would reduce indentation. Not sure if that is necessarily better than an `if else`.

Reply:
Yeah fair. It is only a single line so I wouldn't mind too much but happy to leave it as is.